canvas_manage_course/urls.py

```diff
@@ -15,3 +16,13 @@
     url(r'^not_authorized$', icommons_ui_views.not_authorized, name='not_authorized'),
     url(r'^tool_config$', views.tool_config, name='tool_config'),
 ]
+
+try:
+    if settings.DEBUG:
+        import debug_toolbar
+        urlpatterns += [
+            url(r'^__debug__/', include(debug_toolbar.urls)),
+        ]
+except:
+    pass
```

Comment:
Might be good to have a small comment here so we can remember why we don't care about handling this exception in case we revisit the file.